modules/govuk/manifests/apps/performanceplatform_big_screen_view.pp

```diff
@@ -29,5 +34,17 @@
       ssl_only        => true,
       app_port        => 3058,
     }
+    # Since this is not a full app, we need to ensure env dir exists.
+    file { "/etc/govuk/${app_name}/env.d":
```

Comment:
I think this line needs to be:

```
file { ["/etc/govuk/${app_name}", "/etc/govuk/${app_name}/env.d"]:
```

because the parent directory of the `env.d` directory also isn't created.